presets/4.4/tune/karate/karate_brushless_whoop.txt

```diff
@@ -0,0 +1,144 @@
+#$ TITLE: Karate brushless whoop
+#$ FIRMWARE_VERSION: 4.4
+#$ CATEGORY: TUNE
+#$ STATUS: EXPERIMENTAL
+#$ KEYWORDS: karate, 2s, race, brushless,  whoop, 65mm, sugarK, KarateBrot
```

Comment:
can also add 1S in the keywords since it works for 1S

Reply:
I’m using amend as well 😂